<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт профилей из ИНФИС</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblImportFrom">
       <property name="text">
        <string>импортировать из</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFileName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnView">
       <property name="text">
        <string>Просмотр</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>начать импортирование</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNum">
       <property name="text">
        <string>всего записей в источнике:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbFilterRbLocation">
     <property name="title">
      <string>Справочники для фильтра</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtRbNetFileName"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtRbTypeFileName"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btnSelectFileRbNet">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="btnSelectFileRbType">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnLoadRbNet">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnLoadRbType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblRbNet">
        <property name="text">
         <string>Справочник сетей</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblRbType">
        <property name="text">
         <string>Справочник типов</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnViewRbNet">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Просмотр</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btnViewRbType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Просмотр</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Фильтр</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkNet">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сеть</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cmbNet">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>в</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>д</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>а</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cmbType">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkGTS">
        <property name="text">
         <string>Номер ГТС</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbGTS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="btnLoadGTS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>загрузить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtFileName</tabstop>
  <tabstop>btnSelectFile</tabstop>
  <tabstop>btnView</tabstop>
  <tabstop>edtRbNetFileName</tabstop>
  <tabstop>btnSelectFileRbNet</tabstop>
  <tabstop>btnLoadRbNet</tabstop>
  <tabstop>btnViewRbNet</tabstop>
  <tabstop>edtRbTypeFileName</tabstop>
  <tabstop>btnSelectFileRbType</tabstop>
  <tabstop>btnLoadRbType</tabstop>
  <tabstop>btnViewRbType</tabstop>
  <tabstop>chkNet</tabstop>
  <tabstop>chkType</tabstop>
  <tabstop>chkGTS</tabstop>
  <tabstop>logBrowser</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkNet</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbNet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkType</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGTS</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbGTS</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGTS</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnLoadGTS</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
